code/modules/status_system/statusEffects.dm

```diff
@@ -1526,6 +1526,271 @@
 			H.TakeDamage(zone="All", brute=damage)
 			bleed(H, damage, bleed)
 
+/datum/statusEffect/creeping_dread	//Aka fear of the dark
```

Comment:
move these to a different file, this one doesn't need to be any longer

Reply:
moved to /status_system/wraithEffects.dmi